header/header.go

```diff
@@ -115,13 +113,10 @@ func (eh *ExtendedHeader) Validate() error {
 		return fmt.Errorf("ValidateBasic error on RawHeader at height %d: %w", eh.Height(), err)
 	}
 
-	if eh.RawHeader.Version.App != v1.Version && eh.RawHeader.Version.App != v2.Version {
-		return fmt.Errorf(
-			"app version mismatch, expected: %d or %d, got %d",
-			v1.Version,
-			v2.Version,
-			eh.RawHeader.Version.App,
-		)
+	if eh.RawHeader.Version.App > appconsts.LatestVersion || eh.RawHeader.Version.App == 0 {
```

Comment:
looks like `"github.com/celestiaorg/celestia-app/pkg/appconsts"` isn't imported

Reply:
Nit: keeping `eh.RawHeader.Version.App == 0` at the beginning will make `if` looks a bit simpler.